bigquery/cloud-client/snippets.py

```diff
@@ -33,14 +33,27 @@
 
 
 def list_projects():
-    raise NotImplementedError(
-        'https://github.com/GoogleCloudPlatform/gcloud-python/issues/2143')
+    bigquery_client = bigquery.Client()
+
+    projects = []
+    page_token = None
+
+    while True:
+        results, page_token = bigquery_client.list_projects(
```

Comment:
Is this not using the new iterator thingy? I thought there was a `.pages` or something now.